src/components/StyledList/StyledList.stories.mdx

```diff
@@ -0,0 +1,200 @@
+    className: { control: false },
+    id: { control: false },
+    listItems: { control: false },
+    style: { control: false },
+    textSize: { table: { defaultValue: { summary: "default" } } },
+  }}
+/>
+
+# StyledList
+
+| Component Version | DS Version |
+| ----------------- | ---------- |
+| Added             | `1.3.0`    |
+| Latest            | `1.3.0`    |
+
+## Table of Contents
+
+- [Overview](#overview)
+- [Component Props](#component-props)
+- [Accessibility](#accessibility)
+- [StyledList Children](#styledlist-children)
+- [Font Size](#font-size)
+
+## Overview
+
+<Description of={StyledList} />
+
+export const itemGroups = [
+  'FULL TITLE: "...and for the City of New York"',
+  'ALTERNATIVE TITLES: "...Society of New York delivered at..." + 1 more',
+  'SUBJECTS: "...City of New York, in the" + 2 more',
+];
+
+<Canvas withToolbar>
+  <Story
+    name="StyledList with Controls"
+    args={{
+      className: undefined,
+      id: "nypl-styledlist",
+      listItems: itemGroups,
+      style: "capped",
+      textSize: "default",
+    }}
+  >
+    {(args) => <StyledList {...args} />}
+  </Story>
+</Canvas>
+
+<ArgsTable story="StyledList with Controls" />
+
+## Accessibility
+
+Internally, the `StyledList` component implements native HTML markup for
+unordered lists. When data is passed into the `listItems` data prop, the
+appropriate children `li` HTML elements are used.
+
+Resources:
+
+- [W3C WAI Content Structure Lists](https://www.w3.org/WAI/tutorials/page-structure/content/#lists)
+- [Deque University Appropriate Semantic Markup for Lists](https://dequeuniversity.com/tips/list-markup)
+
+## StyledList Children
+
+### With li children
+
+Pass in `<li>` children elements as you normally would for the `<ul`> element.
+
+```jsx
+<!-- With `li` elements -->
+<StyledList>
+  <li>
+    FULL TITLE: <i>"...and for the <b>City of New York</b>"</i>
+  </li>
+  <li>
+    ALTERNATIVE TITLES: <i>"...Society of <b>New York</b> delivered at..."</i> + 1 more
+  </li>
+  <li>SUBJECTS: <i>"...<b>City of New York</b>, in the"</i> + 2 more</li>
+</StyledList>
+```
+
+<Canvas>
+  <DSProvider>
+    <StyledList>
+      <li>
+        FULL TITLE:{" "}
+        <i>
+          "...and for the <b>City of New York</b>"
+        </i>
+      </li>
+      <li>
+        ALTERNATIVE TITLES:{" "}
+        <i>
+          "...Society of <b>New York</b> delivered at..."
+        </i>{" "}
+        + 1 more
+      </li>
+      <li>
+        SUBJECTS:{" "}
+        <i>
+          "...<b>City of New York</b>, in the"
+        </i>{" "}
+        + 2 more
+      </li>
+    </StyledList>
+  </DSProvider>
+</Canvas>
+
+### With listItems prop
+
+If `<li>` elements cannot be passed as children to the `StyledList` component,
+an array with data can be passed into the `listItems` prop. Note: either
+children elements or the `listItems` prop should be passed but not both.
+
+For data in the `listItems` prop, the `StyledList` component will render the
```

Comment:
Maybe add a reminder here that the data can be a string or a JSX Element.

Data in the `listItems` prop array can be JSX Elements or strings. The `StyledList` component will render the data as `li` elements, so there is no need to add them in the `listItems` array.